README.md

```diff
@@ -11,24 +11,26 @@ This repository holds a minimal installer for conda specific to conda-forge. It
 Miniforge installers are available here: https://github.com/conda-forge/miniforge/releases
 
 #### Miniforge3
+
 Latest installers with python 3.8 in the base environment
 
-- [linux-aarch64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh) (also called `arm64`)
-- [linux-ppc64le](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-ppc64le.sh) (also called `POWER8/9`)
-- [linux-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh) (also called `amd64`)
-- [osx-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh)
-- [osx-arm64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh) (Apple Silicon)
-- [win-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe)
+- [`linux-aarch64`](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh) (also called `arm64`)
```

Comment:
Do we want monospace fonts for these?

Reply:
I've seen in the document, in the testing chapter.

> architecture is `aarch64`, the constructed installer is tested against:

This is why I've applied the same convention. Tell me I've no firm opinion on the matter. It just would be nice to apply the same convention.